lib/workers/branch/index.spec.ts

```diff
@@ -1193,11 +1214,23 @@ describe('workers/branch/index', () => {
         } as StatusResult);
 
       fs.outputFile.mockReturnValue();
-      fs.readFile
-        .mockResolvedValueOnce(Buffer.from('modified file content'))
-        .mockResolvedValueOnce(Buffer.from('this file will not exists'))
-        .mockResolvedValueOnce(Buffer.from('modified file content again'))
-        .mockResolvedValueOnce(Buffer.from('this file was once deleted'));
+      fsUtil.readLocalFile
+        .mockResolvedValueOnce(Buffer.from('modified file content') as never)
```

Comment:
Do we really need that cast? 😕

Reply:
@viceice this cast is needed to avoid TypeScript errors. I had to add it.